index.html

```diff
@@ -17,7 +17,7 @@
     <header>
       <nav class="nav">
         <h1 class="my-logo">My logo</h1>
-        <img src="assets/Unionnavbar.png" class="navbar">
+        <img src="assets/Unionnavbar.png" alt="nav-bar" class="navbar">
         <ul class="nav-list">
           <li class="nav-item1">Portfolio</li>
           <li class="nav-item2">About</li>
```

Comment:
- [ ] Kindly make use of nav tag so that the link can be accessible by screenreaders and tab key.

Reply:
@Olamarx we already use the nav tag in the header just check it again, thanks